src/main/java/md2k/mCerebrum/cStress/features/ECGFeatures.java

```diff
@@ -503,18 +500,16 @@ private void filterPeaks(DataPointStream rrAverage, DataPointStream temp1, DataP
                             Rpeak_temp1.add(new DataPoint(0, 0.0));
                         }
                         Rpeak_temp1.set(c1, peaks.data.get(c2.get(c1 - 1) + searchback_array_inrange_index.get(searchback_max_index)));
-                        sig_lev.add(new DataPoint(Rpeak_temp1.get(c1).timestamp, Smoothing.ewma(Rpeak_temp1.get(c1 - 1).value, sig_lev.getLatestValue(), AUTOSENSE.EWMA_ALPHA)));
+                        sig_lev.add(new DataPoint(Rpeak_temp1.get(c1).timestamp,0.125*Rpeak_temp1.get(c1).value+0.875*sig_lev.getLatestValue()));
```

Comment:
Why my the Smoothing.ewma function removed.  Some of this code seems like a retroactive pull of logic from an older commit.